### ISSUE TITLE
Addon settings in Options screen are missing

### ISSUE THREAD

**Issue by jgleigh** [rursache/InFlight-TWW#6]
Open the InFlight section of addon settings and it's just a blank screen.

**Comment by MrDarkflame** [comment]
Was able to solve this by changing InFlight_Load.lua
```
local category = Settings.RegisterCanvasLayoutCategory(self, "InFlight")
Settings.RegisterAddOnCategory(category)
```
to
```
category, layout = Settings.RegisterCanvasLayoutCategory(panel, panel.name, panel.name)
category.ID = panel.name
Settings.RegisterAddOnCategory(category);
```

**Comment by rursache** [comment]
fixed in v11.0.2
https://github.com/rursache/InFlight-WW/releases/tag/v11.0.2
